python/cli/prelude_cli/views/build.py

```diff
@@ -312,15 +312,17 @@ def delete_detection(controller, detection):
 
 @build.command('create-threat-hunt')
 @click.argument('name')
+@click.option('-c', '--control', help='', required=True, type=click.Choice([c.name for c in Control]), case_sensitive=False)
 @click.option('-q', '--query', help='Threat hunt query', required=True, type=str)
 @click.option('-t', '--test', help='ID of the test this threat hunt query is for', required=True, type=str)
 @click.option('--id', default=None, type=str)
 @click.pass_obj
 @handle_api_error
-def create_threat_hunt(controller, name, query, test, id):
+def create_threat_hunt(controller, name, control, query, test, id):
     """ Create a threat hunt query """
     with Spinner(description='Creating new threat hunt'):
         t = controller.create_threat_hunt(
+            partner=Control[control.upper()],
```

Comment:
the upper is unnecessary

Reply:
wouldn't setting case_sensitive=false mean people could pass through 'crowdstrike' and it would be a valid choice?

Reply:
https://github.com/preludeorg/libraries/blob/2c67c59ca489806775e6379776a9e8f69ee28662/python/sdk/prelude_sdk/models/codes.py#L9

yep

Reply:
coooool thats convenient